### PR TITLE
Embed updated review

### PR DESCRIPTION
**Pull request by cldougl**

### CODE REVIEW

File: _posts/fundamentals/2015-04-23-embed-plotly-graphs.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ popularity: featured
 actioncall: Embed a Graph
 
 steps:
- - title: Embed Plotly Graphs in A Blog or Website
+ - title: Embed Plotly Graphs in a Blog or Website
    sub-steps:
     - copy: "When you embed a Plotly graph, it means you’re sharing your graph, your data and the code that describes your graph all in one place. With our interactive features, your readers have a whole new way to engage with your work. When you update a Plotly graph, the graph automatically updates on your blog or website. No need for manual updates!"
```
Comment (cldougl) [inline _posts/fundamentals/2015-04-23-embed-plotly-graphs.md]: mention you can embed any plotly graph (i.e. a graph created with the api or the workspace)

File: _posts/fundamentals/2015-04-23-embed-plotly-graphs.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ popularity: featured
...
     - copy: "Plotly’s interactive charts add impact to your blog, webpage, online notebook, IPython notebook, and lesson plans. We’re in [Wired Magazine](http://www.wired.com/2014/08/lego-cost), [The Washington Post](http://www.washingtonpost.com/blogs/wonkblog/wp/2013/06/14/do-low-taxes-on-the-rich-leave-the-middle-class-with-lower-wages/), and the [National Post](http://sports.nationalpost.com/2014/08/02/how-does-p-k-subbans-new-contract-stack-up-against-other-elite-nhl-defencemen)."
```
Comment (cldougl) [inline _posts/fundamentals/2015-04-23-embed-plotly-graphs.md]: I'd remove lesson plans and just list the places you can embed that are discussed in this post.

File: _posts/fundamentals/2015-04-23-embed-plotly-graphs.md [hunk 3]
```diff
@@ -14,16 +14,16 @@ actioncall: Embed a Graph
...
-    - copy: "Plotly’s interactive charts add impact to your blog, webpage, online notebook, IPython notebook, and lesson plans. We’re in [Wired Magazine](http://www.wired.com/2014/08/lego-cost), [The Washington Post](http://www.washingtonpost.com/blogs/wonkblog/wp/2013/06/14/do-low-taxes-on-the-rich-leave-the-middle-class-with-lower-wages/), and the [National Post](http://sports.nationalpost.com/2014/08/02/how-does-p-k-subbans-new-contract-stack-up-against-other-elite-nhl-defencemen)."
+    - copy: "When you embed a Plotly graph, it means you’re sharing your graph, your data and the code that describes your graph all in one place. You can embed any Plotly graph, you're using the online workspace, or the API. With our interactive features, your readers have a whole new way to engage with your work. When you update a Plotly graph, the graph automatically updates on your blog or website. No need for manual updates!"
```
Comment (cldougl) [inline _posts/fundamentals/2015-04-23-embed-plotly-graphs.md]: @ellecj not sure this sentence is grammatically correct:
> You can embed any Plotly graph, you're using the online workspace, or the API.

Maybe: You can embed any Plotly graph. The embedding process is the same whether you're creating graphs from the online workspace, or using one of Plotly's APIs.

File: _posts/fundamentals/2015-04-23-embed-plotly-graphs.md [hunk 4]
```diff
@@ -14,16 +14,16 @@ actioncall: Embed a Graph
...
          
  - title: From Your Workspace
    sub-steps:
-    - copy: "To share from your own workspace, go to SHARE on the left-hand side."
+    - copy: "To share a plot from your own workspace, go to SHARE on the left-hand side."
       img: "![Workspace share option](../static/images/how-to-save-export-and-share/new share main.png)"
-    - copy: "The Share popup will display a link that you can add to a website and you have the option of embedding your plot as an HTML snippet or iframe. Make sure that the graph is set to 'Public' in order to generate the code."
+    - copy: "The Share popup will display a link that you can add to a website and you have the option of embedding your plot as an HTML snippet or iframe. Make sure that the graph is set to 'Public' in order to generate the code. It can be set to secret as well; you just need to share the link."
```
Comment (cldougl) [inline _posts/fundamentals/2015-04-23-embed-plotly-graphs.md]: @ellecj this still isn't quite correct - the plot doesn't need to be public to generate the embed code... you can generate the embed code with a secret graph then the share_key is automatically added to the plot url in the embed code:
screen shot 2016-10-25 at 1 56 35 pm

File: _posts/fundamentals/2015-04-23-embed-plotly-graphs.md (hunk 3 above)
Comment (ellecj) [inline _posts/fundamentals/2015-04-23-embed-plotly-graphs.md]: The word "whether" is missing!

**Pull request merged**